lib/bookingsync/api/client/rentals_contents_overrides.rb

```diff
@@ -0,0 +1,72 @@
+module BookingSync::API
+  class Client
+    module RentalsContentsOverrides
+      # List rentals_contents_overrides
+      #
+      # Returns all rentals contents overrides for the current account.
+      # @param options [Hash] A customizable set of options.
+      # @option options [Array] fields: List of fields to be fetched.
+      # @return [Array<BookingSync::API::Resource>] Array of rentals_contents_overrides.
+      #
+      # @example Get the list of rentals contents overrides for the current account
+      #   rentals_contents_overrides = @api.rentals_contents_overrides
+      #   rentals_contents_overrides.first.description.en # => "Overriden English description"
+      # @see http://developers.bookingsync.com/reference/endpoints/rentals_contents_overrides/#list-rentals-contents-overrides
+      def rentals_contents_overrides(options = {}, &block)
+        paginate :rentals_contents_overrides, options, &block
+      end
+
+      # Get a single rentals_contents_override
+      #
+      # @param rentals_contents_override [BookingSync::API::Resource|Integer]
+      # rentals_contents_override or ID of the rentals_contents_override.
+      # @return [BookingSync::API::Resource]
+      # @see http://developers.bookingsync.com/reference/endpoints/rentals_contents_overrides/#get-a-single-rentals-content-override
+      def rentals_contents_override(rentals_content_override)
+        get("rentals_contents_overrides/#{rentals_content_override}").pop
```

Comment:
No specs for accepting resource as argument, only for ID. No spec for what it's returning.

Reply:
Accepting resource as argument is already tested inside resource by checking that "#{resource}" == resource.id.to_s